test_assets/portalnet/bridge_data.yaml

```diff
@@ -0,0 +1,2 @@
+- content_key: 0x006251d65b8a8668efabe2f89c96a5b6332d83b3bbe585089ea6b2ab9b6754f5e9
```

Comment:
Well, if we're going to switch to yaml to support comments, let's use some here! Something like at least the block number and also a blurb about where & how this test data is used would be a good start

Reply:
I don't know what block number you generated this data from

Reply:
I found the block number so Sure I will add a comment on the blocks height